react-hot-button/vite-project/src/App.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import './App.css';
+import { HotButton } from './HotButton';
+const backgroundColor = [
+  'rgb(155,113,139)',
+  'rgb(103,78,167)',
+  'rgb(224,102,102)',
+  'rgb(216,156,94)',
+  'rgb(255,255,0)',
+  'rgb(255,255,255)',
+];
+export default function App() {
+  const [count, setCount] = useState(0);
+  function handleClick() {
+    setCount(count + 1);
+  }
+  const label = `${count} Clicks`;
+  let index = 0;
+  switch (true) {
```

Comment:
I've seen logic like this before but I personally don't find it as clear as a simple series of if/else statements.